config/resolvers/resolvers-deployment.yaml

```diff
@@ -106,6 +106,9 @@ spec:
           value: "https://artifacthub.io/"
         - name: TEKTON_HUB_API
           value: "https://api.hub.tekton.dev/"
+        volumeMounts:
+          - name: tmp-clone-volume
+            mountPath: "/tmp"
```

Comment:
Mounting a directory at `/tmp` is necessary because the security context sets the root filesystem as read-only, and since we're using `git clone` we have to clone into the filesystem. 

Additionally, using an empty-dir volume as the `/tmp` directory allows us to configure the size of the directory in alignment with the pod's memory requests.